src/examples/PersistenceExample/Program.cs

```diff
@@ -16,16 +16,46 @@ static void Main(string[] args)
             {
                 //BasicUsage(system);
 
-                // FailingActorExample(system);
+                //FailingActorExample(system);
 
-                SnapshotedActor(system);
+                //SnapshotedActor(system);
 
                 //ViewExample(system);
 
+                GuaranteedDelivery(system);
+
                 Console.ReadLine();
             }
         }
 
+        private static void GuaranteedDelivery(ActorSystem system)
+        {
+            Console.WriteLine("\n--- GUARANTEED DELIVERY EXAMPLE ---\n");
+            var delivery = system.ActorOf(Props.Create(()=> new DeliveryActor()),"delivery");
+
+            var deliverer = system.ActorOf(Props.Create(() => new GuaranteedDeliveryExampleActor(delivery.Path.ToString())));
```

Comment:
You don't need to stringify `ActorPath`, since it's parsed later in the code. Just push it through.